pkgs/tools/security/pinentry-mac/default.nix

```diff
@@ -17,6 +17,10 @@ stdenv.mkDerivation rec {
     mv build/Release/pinentry-mac.app $out/Applications
   '';
 
+  passthru = {
+    binaryPath = "Applications/pinentry-mac.app/Contents/MacOS/pinentry-mac";
```

Comment:
Did not appreciate existence of a separate qt5 expression. In that case it's 2 places to hardcode things either way. So it's fine to keep it as is, I guess.